src/main/java/reposense/parser/GroupConfigCsvParser.java

```diff
@@ -0,0 +1,86 @@
+package reposense.parser;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.apache.commons.csv.CSVRecord;
+
+import reposense.model.FileType;
+import reposense.model.GroupConfiguration;
+import reposense.model.RepoLocation;
+
+/**
+ * Container for the values parsed from {@code groups-config.csv} file.
+ */
+public class GroupConfigCsvParser extends CsvParser<GroupConfiguration> {
+    public static final String GROUP_CONFIG_FILENAME = "groups-config.csv";
+
+    /**
+     * Positions of the elements of a line in groups-config.csv config file
+     */
+    private static final int LOCATION_POSITION = 0;
+    private static final int GROUP_NAME_POSITION = 1;
+    private static final int FILES_GLOB_POSITION = 2;
+
+    public GroupConfigCsvParser(Path csvFilePath) throws IOException {
+        super(csvFilePath);
+    }
+
+    /**
+     * Gets the list of positions that are mandatory for verification.
+     */
+    @Override
+    protected int[] mandatoryPositions() {
+        return new int[] {
+            LOCATION_POSITION, GROUP_NAME_POSITION, FILES_GLOB_POSITION,
+        };
+    }
+
+    /**
+     * Processes the csv file line by line and adds created {@code Group} into {@code results}.
+     */
+    @Override
+    protected void processLine(List<GroupConfiguration> results, CSVRecord record) {
+        String location = get(record, LOCATION_POSITION);
+        String groupName = get(record, GROUP_NAME_POSITION);
+        List<String> globList = getAsList(record, FILES_GLOB_POSITION);
+
+        GroupConfiguration groupConfig = null;
+        try {
+            groupConfig = findMatchingGroupConfiguration(results, location);
+        } catch (InvalidLocationException e) {
+            e.printStackTrace();
+        }
```

Comment:
What happens when `InvalidLocationException` is thrown?

`groupConfig` is `null` and the following lines will fail?

Reply:
That line should fail and not be parsed in.

Have corrected this part of the code.